packages/terra-date-input/src/DateInput.jsx

```diff
@@ -499,7 +510,7 @@ class DateInput extends React.Component {
           disabled={this.props.disabled}
           aria-describedby={`${this.formatDescriptionId} ${this.props.monthAttributes['aria-describedby']}`}
         >
-          <option value="" hidden>{this.props.intl.formatMessage({ id: 'Terra.date.input.monthPlaceholder' })}</option>
+          <option value="">{this.props.intl.formatMessage({ id: 'Terra.date.input.monthPlaceholder' })}</option>
```

Comment:
In Firefox when no month is initially selected in the month select, all the months appear as disabled and italicized similar to this `- Select -` placeholder but they are actually active.

![Screen Shot 2020-10-13 at 1 33 04 PM](https://user-images.githubusercontent.com/22618655/95901929-aff12900-0d59-11eb-93e9-0998bab41281.png)

Reply:
Looks like this [line](https://github.com/cerner/terra-framework/blob/4fa0427ed5c424247a12cc66c4b711d8943b09b5/packages/terra-date-input/src/DateInput.jsx#L478) is causing this behaviour in firefox. As a workaround thought of adding this style for all options except the placeholder one, which is similar to this [block](https://github.com/cerner/terra-framework/blob/main/packages/terra-date-input/src/DateInput.module.scss#L232-L237).
```  
    option:not([value='']) {
      color: transparent;
      font-style: normal;
      text-shadow: 0 0 0 #1c1f21;
    }
```

Reply:
Done 9f1f78c